SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc

```diff
@@ -0,0 +1,71 @@
+The OBX-18 field shall contain the equipment (or device) identifier on the MDS level and/or the measurement module identifier of the equipment on the VMD level.
+
+The table <<ref_tbl_dec_obx18_mds_mapping>> defines the mapping of the SDC MDS meta data to the data fields of the HL7 data type *EI* used in the OBX-18 field.
+
+he table <<ref_tbl_dec_obx18_vmd_mapping>> defines the mapping of the SDC VMD information to the data fields of the HL7 data type *EI* used in the OBX-18 field.
+
+[#ref_tbl_dec_obx18_mds_mapping]
+.OBX-18 Equipment Instance Identifier Mapping - MDS level

```

Comment:
This mapping heavily relies on the MDS's UDI DeviceIdentifier to be available. In BICEPS, the whole UDI element is optional, so is the DeviceIdentifier. Is the non-existence of the UDI covered herein - or is it just mandatory requirement in order to be conformant to SDPi at that point?

Reply:
It is mandatory to identify the device. Hopefully, we can discuss this in the week Nov. 28th when we meet in Luebeck again.

Reply:
For the OBX18 mapping, I added a note for reviews with a request for comment.